src/main/kotlin/no/nav/familie/ef/iverksett/arena/VedtakhendelseMapper.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.familie.ef.iverksett.arena
+
+import no.nav.familie.ef.iverksett.iverksetting.domene.Iverksett
+import no.nav.melding.virksomhet.vedtakhendelser.v1.vedtakhendelser.WSApplikasjoner
+import no.nav.melding.virksomhet.vedtakhendelser.v1.vedtakhendelser.WSAvslutningsstatuser
+import no.nav.melding.virksomhet.vedtakhendelser.v1.vedtakhendelser.WSBehandlingstema
+import no.nav.melding.virksomhet.vedtakhendelser.v1.vedtakhendelser.WSVedtakHendelser
+import java.text.SimpleDateFormat
+import java.util.*
+import javax.xml.datatype.DatatypeFactory
+
+
+fun mapIverksettTilVedtakHendelser(iverksett: Iverksett): WSVedtakHendelser {
+    val wsVedtakHendelser = WSVedtakHendelser()
```

Comment:
Burde kanskje notert det i PR for tjenestespesifikasjoner, med prefix WS = WebService, som kanskje blir litt missvisende her :P småpirk, du kan ju vurdere hva du syns

Reply:
Enig, pr def blir det feil å kalle det for WebService. Lagde eget objekt med navn uten WS som prefix.